prompts/all.json

```diff
@@ -54,6 +54,17 @@
     }
   ],
   "tabular": [
+    {
+      "id": "input-output-pairs",
+      "label": "Create a question-and-answer dataset based on a source text",
+      "value": [
+        "From the source text below, create a dataset with the following columns:",
```

Comment:
**question**: I know @hhpang has been experimenting with setting the number of rows in the prompt value (30 in the deleted example). Do we need to keep doing that here?

Reply:
Ah, good catch! It's just to guide/hint to the user they can request num row through the prompt. This will generate the default 50 rows which is fine too.